app/views/request_for_comments/show.html.slim

```diff
@@ -1,7 +1,9 @@
 .list-group
   h4#exercise_caption.list-group-item-heading data-exercise-id=@request_for_comment.exercise.id data-rfc-id=@request_for_comment.id
     - if @request_for_comment.solved?
-      span.fa-solid.fa-check.me-2 aria-hidden='true'
+      span.fa-solid.fa-check.text-success.me-2 aria-hidden='true'
+    - elsif @request_for_comment.full_score_reached
```

Comment:
```suggestion
    - elsif @request_for_comment.full_score_reached?
```

Sounds like a boolean condition, so we can indicate it accordingly. Didn't look it up.

Reply:
Yes, that's a boolean condition. I verified the change locally, and as expected and usual in these cases, the `?` can be appended.